addons/im_livechat/models/discuss_channel.py

```diff
@@ -96,6 +96,37 @@ def _gc_empty_livechat_sessions(self):
     def execute_command_history(self, **kwargs):
         self._bus_send("im_livechat.history_command", {"id": self.id})
 
+    def execute_command_bot(self, bot_id, **kwargs):
+        if not self.livechat_active:
+            # To-do: remove this after livechat visitor leave has been adapted to use action_unfollow
+            return
+        chatbot_script = self.env['chatbot.script'].browse(bot_id)
+        if not self or not chatbot_script.exists():
+            self._send_transient_message(self.env.user.partner_id, _("Please select a chatbot"))
+            return
+        if self.livechat_operator_id != chatbot_script.operator_partner_id:
+            operator_partner = self.env['chatbot.script'].search(
+                [('operator_partner_id', '=', self.livechat_operator_id.id)],
```

Comment:
:pick: We try to enforce double quotes for strings in new code. You might want to install `ruff` to format the code you wrote to help with these validations.